lib/WebpackOptionsDefaulter.js

```diff
@@ -140,6 +140,8 @@ class WebpackOptionsDefaulter extends OptionsDefaulter {
 				// Lazy load the uglifyjs plugin
 				const UglifyJsPlugin = require("uglifyjs-webpack-plugin");
 				new UglifyJsPlugin({
+					cache: true,
+					parallel: true,
 					sourceMap: options.devtool && /source-?map/.test(options.devtool)
```

Comment:
Should `uglifyjs-webpack-plugin` really produce source maps when running in `production` mode ? It slows down the production build and the source maps are often 'broken' because of e.g DCE by `uglify` and webpack's 'line-to-line' source maps. Most issues in `uglifyjs-webpack-plugin` nowadays are related to `sourceMap` which won't be easy to fix I assume and the best 'fix' is to simply not uglify during 'development' :) imho...

Reply:
Source maps in production are common these days. Browsers are smart enough to not download them if your devtools aren't open & for reference, at least in the framework ecosystem I work in, prod source maps are an expectation.